setup.cfg

```diff
@@ -25,18 +25,36 @@ classifiers =
     Programming Language :: Python :: 3.6
     Topic :: Office/Business
 
-install_requires =
-    django>=2.2.10,<2.2.1024
-    redis>=3.4.1,<3.4.1024
-    Pillow>=7.0.0,<7.0.1024
-    python-dateutil>=2.8.1,<2.8.1024
-    bleach>=3.1.1,<3.1.1024
-    django-formtools==2.2
 
 [options]
 zip_safe = False
 include_package_data = True
 packages = find:
+install_requires =
+    django >=2.2.10,<2.2.1024
+    redis >=3.4.1,<3.4.1024
+    Pillow >=7.0.0,<7.0.1024
+    python-dateutil >=2.8.1,<2.8.1024
+    bleach >=3.1.1,<3.1.1024
+    django-formtools ==2.2
+    xlrd >=1.2.0,<1.2.1024
+    xlwt >=1.3.0,<1.3.1024
+
+[options.extras_require]
+dev=
+    coverage >=5.1,<5.1.1024
+    flake8 >=3.8.2,<3.8.1024
+    isort >=4.3.21,<4.3.1024
+    parameterized >=0.7.1,<0.7.1024
+    tblib >=1.6.0,<1.6.1024
```

Comment:
please add `django-extensions` :pray:

Reply:
I agree it's really usefull and we should add it, but it's not the purpose of this PR

Reply:
This line is already in the current requirements-dev.txt:
django-extensions >=2.2.9,<2.2.1024